### PR TITLE
[F] ProgressBar

### PR DESCRIPTION
**Pull request by alexgoff**
Progress Bar for investigations, pulled from investigation-static with minor adaptations.

### CODE REVIEW

File: packages/epo-react-lib/jest-setup.ts [hunk 1]
```diff
@@ -1,2 +1,22 @@
 import "@testing-library/jest-dom";
 import "jest-canvas-mock";
+
+beforeAll(() => {
```
Comment (alexgoff) [inline packages/epo-react-lib/jest-setup.ts:4]: some tests in other areas were failing, grabbed this out of the widgets lib

File: packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  max = 100,
+  value,
+  displayValue,
+  labelledById,
+  describedById,
+  children,
+  className,
+}) => {
+  const safeValue = Math.min(Math.max(value, min), max);
+  const isIndeterminate = isNaN(safeValue);
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx:25]: we can use indeterminate status later for any sort of loaders

File: packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  min = 0,
+  max = 100,
+  value,
+  displayValue,
+  labelledById,
+  describedById,
+  children,
+  className,
+}) => {
+  const safeValue = Math.min(Math.max(value, min), max);
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx:24]: make sure value is always bounded between min and max for ARIA compatibility

File: packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <Styled.ProgressBarContainer className={className}>
+      <Styled.ProgressBar
+        role="progressbar"
+        aria-valuemin={min}
+        aria-valuemax={max}
+        aria-valuenow={isIndeterminate ? undefined : safeValue}
+        aria-labelledby={labelledById}
+        aria-describedby={describedById}
+        aria-valuetext={displayValue}
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/ProgressBar/ProgressBar.tsx:36]: if a display value is set, this will override the default readout

**Review by blnkt — APPROVED** [review]

**Pull request merged**